frontend/src/components/WofCard.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+    <div class="card">
+        <slot/>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'WofCard'
+}
+</script>
+
+<style lang="less">
+@import './common.less';
+
+.card {
+    display: flex;
+    padding: 1em;
+    background-color: @secondary-color;
+    border-radius: 20px;
+    width: min-content;
+    height: min-content;
+}
```

Comment:
Do we want this to be scoped?

Reply:
We will use BEM naming strategy and also this is kinda common, library component so here it's ok